frontend/pages/admin/IntegrationsPage/cards/CertificateAuthorities/components/AddCertAuthorityModal/helpers.tsx

```diff
@@ -42,7 +42,7 @@ const PRIVATE_KEY_NOT_CONFIGURED_ERROR = (
     Private key must be configured.{" "}
     <CustomLink
       text="Learn more"
-      url="https://learn-more-about/fleet-server-private-key"
+      url="https://fleetdm.com/learn-more-about/fleet-server-private-key"
```

Comment:
For consistency, can you update this to
`import { LEARN_MORE_ABOUT_BASE_LINK } from "utilities/constants";`

Reply:
nice. updated.